### DISCUSSION TITLE
Anyone knows how to do easter egg on Fegefeuer?

### DISCUSSION THREAD

**Discussion by ToonLink0001** [nzp-team/nzportable#704] [General]
hey so I found 2 teddys and Idk if there is a third one or if there is an easter egg song, dose anyone know?

**Comment by Reznar705** [comment]
where are the two teddy? i know ones in the starting room

**Reply by ToonLink0001** [reply]
The second one is in pack a punch

**Comment by Reznar705** [comment]
oh yeah i remember seeing that
